### DISCUSSION TITLE
I am getting undefined error in the node module of the library

### DISCUSSION THREAD

**Discussion by ali-iftikhar007** [antvis/antvis.github.io#172] [Q&A] [answered]
`Uncaught TypeError: Cannot read properties of undefined (reading 'name')
    at View.n.onCanvasEvent (view.js:60:1)
    at doEmit (index.js:60:1)
    at EventEmitter.emit (index.js:64:1)
    at canvas.js:223:1
    at sentryWrapped (helpers.js:91:1)
` 

I am using the library to display the plots on my web app and I am using Sentry for bug tracking. Recently, I saw that Sentry has reported 60K errors related to this error. (Mentioned above) 

What can I do tow fix that? 

Currently I am using this version -> "@ant-design/plots": "^1.2.5",

**Comment by nikojxie** [comment]
I encountered the same problem

**Reply by ali-iftikhar007** [reply]
Any fix you found? @nikojxie

**Reply by hustcc** [reply, marked as answer]
Update the dependent, will have fixed the issue yesterday. ensure the version of @antv/g-canvas is 0.5.16.
